General/CodeSmells/Comments/Src/Comments.Problem/Good/MagicValues.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Comments.Problem.Bad;
+
+namespace Comments.Problem.Good
+{
+    /// <summary>
+    /// Solution to <see cref="Bad.Salary"/>
+    /// Instead of magic numbers and comments prefer to use named constants.
+    /// </summary>
+    public class Salary
+    {
+        public double Calculate(float main)
+        {
+            // coeficient * multiplier * taxes - mandatory pay.
```

Comment:
Following the logic here: 
Coefficient is the first term (1), multiplier is the second (0.99) and taxes is the third (main) and mandatory pay (final term)? Maybe more realistic example values might be better. Also - what is it calculating? Salary? Tax Paid? Not clear.